database/database.go

```diff
@@ -1,24 +1,20 @@
 package database
```

Comment:
I haven't found a place where we ensure the collections? That'd be the place where you also ensure they have proper indices.

Reply:
The driver will probably have some examples in the repo, or in the documentation.

Reply:
https://github.com/mongodb/mongo-go-driver/blob/master/examples/documentation_examples/examples.go

Reply:
F/U: https://github.com/NebulousLabs/skynet-accounts/issues/18